### PR TITLE
feat: launch desktop app

### PR DESCRIPTION
**Pull request by 2fd**
- if the url has the position param it try to open the desktop version (even in macOS)
- if detect that the desktop version is installed it prevent the autologin
- if window was detected shows a message to download the client

### CODE REVIEW

File: src/integration/desktop.ts [hunk 1]
```diff
@@ -59,3 +61,37 @@ export const initializeDesktopApp = callOnce(() => {
     console.log('Electron found')
   }
 })
+
+export async function launchDesktopApp() {
```
Comment (menduz) [inline src/integration/desktop.ts:65]: Can you add a comment to explain what this function should do? I read it three times and it is hard to follow, I'm not sure what it does

File: src/utils/parsePosition.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  if (!Number.isFinite(x) || !Number.isFinite(y)) {
+    return null
+  }
+
+  if(!isInsideWorldLimits(x,y)) {
+    return null
+  }
+
+  return [x,y] as const
+}
```
Comment (fzavalia) [inline src/utils/parsePosition.ts:18]: EOL

File: src/integration/desktop.ts (hunk 1 above)
Comment (2fd) [reply src/integration/desktop.ts:70]: @menduz could you check the new version?

**Comment by 2fd** [comment]
@fzavalia could you check it?

**Review by fzavalia — APPROVED** [review]
Liked the comments on the `launchDesktopApp`, makes it much more clear.
The touching message with the login item has been solved.
LGTM!
Note: I don't have windows so I can't manually confirm that everything is working as intended, yet the code seems correct. Would be nice if a windows user or at least someone with a windows VM can test it out.

File: src/integration/desktop.ts [hunk 3]
```diff
@@ -59,3 +60,57 @@ export const initializeDesktopApp = callOnce(() => {
...
+ * and return a boolean that represents if a loss of focus was detected on the current window
+ * (assuming it was due to the interaction generated by the desktop version)
+ */
+export async function launchDesktopApp() {
+  // prevent launch for desktop and mobile
+  if (isElectron() || isMobile()) {
+    return false
+  }
+
+  // build custom protocal target usign current url `position` and `realm`
```
Comment (fzavalia) [inline src/integration/desktop.ts:75]:
```suggestion
  // build custom protocol target using current url `position` and `realm`
```

File: src/integration/desktop.ts (hunk 1 above)
Comment (menduz) [reply src/integration/desktop.ts:69]: so nice, much better, many thanks

**Review by menduz — APPROVED** [review]

**Review by nachomazzara — APPROVED** [review]
